codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "builtin/gen/bindata.go"
```

Comment:
```suggestion
  - "builtin/gen/bindata.go"
  - "tracers/**/gen_*.go"
```

Reply:
the tracers files are generated right ?

Reply:
do we have any other places with generated files I think we can add them :)